packages/aws-client/package.json

```diff
@@ -46,32 +46,32 @@
   "author": "Cumulus Authors",
   "license": "Apache-2.0",
   "dependencies": {
-    "@aws-sdk/client-api-gateway": "^3.499.0",
-    "@aws-sdk/client-cloudformation": "^3.447.0",
-    "@aws-sdk/client-cloudwatch-events": "^3.447.0",
-    "@aws-sdk/client-dynamodb": "^3.447.0",
-    "@aws-sdk/client-dynamodb-streams": "^3.447.0",
-    "@aws-sdk/client-ec2": "^3.447.0",
-    "@aws-sdk/client-ecs": "^3.447.0",
-    "@aws-sdk/client-elasticsearch-service": "^3.529.1",
-    "@aws-sdk/client-kinesis": "^3.447.0",
-    "@aws-sdk/client-kms": "^3.447.0",
-    "@aws-sdk/client-lambda": "^3.529.1",
-    "@aws-sdk/client-s3": "^3.447.0 <3.614.2",
```

Comment:
If these are newer changes to avoid a vulnerability or something, this may be wrong. We'll see what CI says.